galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager/src/test/java/dev/galasa/core/manager/internal/CoreManagerTest.java

```diff
@@ -94,6 +98,56 @@ public void testCoreManagerGenerate() throws ManagerException, ResourceUnavailab
         
         @StoredArtifactRoot
         public Path root;
+        
+    }
+    
+    
+    public static class TestClassNoAnnotation {
+        
+    }
+    
+    @TestAreas({})
+    @Tags({})
+    public static class TestClassEmptyAnnotation {
+        
+    }
+    
+    
+    @Test
```

Comment:
Happy to be proved wrong here - BUT doe we need to reset the static class between each test method - or will JUnit take care of that?